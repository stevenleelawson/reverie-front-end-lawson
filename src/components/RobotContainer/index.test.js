import React from 'react';
import { RobotContainer } from './index';
import { shallow } from 'enzyme';

describe('RobotContainer', () => {
  it('should match the snapshot', () => {
    const mockRobot =[{
      date_added: '10/27/70',
      first_active: '05/10/12',
      current_name: 'Bender',
      height: '300 in',
      weight: '400 lbs',
      intelligence_metric: 5
    }];

    const wrapper = shallow(<RobotContainer data={mockRobot}/>)
    expect(wrapper).toMatchSnapshot()
  });
});
