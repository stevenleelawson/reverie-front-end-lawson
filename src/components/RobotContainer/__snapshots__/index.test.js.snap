// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RobotContainer should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    data={
        Array [
            Object {
              "current_name": "Bender",
              "date_added": "10/27/70",
              "first_active": "05/10/12",
              "height": "300 in",
              "intelligence_metric": 5,
              "weight": "400 lbs",
            },
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Unknown
          current_name="Bender"
          date_added="10/27/70"
          first_active="05/10/12"
          height="300 in"
          id={undefined}
          intelligence_metric={5}
          removeRobot={undefined}
          updateRobot={undefined}
          weight="400 lbs"
/>,
      ],
      "className": "robot-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "Bender0",
        "nodeType": "function",
        "props": Object {
          "current_name": "Bender",
          "date_added": "10/27/70",
          "first_active": "05/10/12",
          "height": "300 in",
          "id": undefined,
          "intelligence_metric": 5,
          "removeRobot": undefined,
          "updateRobot": undefined,
          "weight": "400 lbs",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Unknown
            current_name="Bender"
            date_added="10/27/70"
            first_active="05/10/12"
            height="300 in"
            id={undefined}
            intelligence_metric={5}
            removeRobot={undefined}
            updateRobot={undefined}
            weight="400 lbs"
/>,
        ],
        "className": "robot-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "Bender0",
          "nodeType": "function",
          "props": Object {
            "current_name": "Bender",
            "date_added": "10/27/70",
            "first_active": "05/10/12",
            "height": "300 in",
            "id": undefined,
            "intelligence_metric": 5,
            "removeRobot": undefined,
            "updateRobot": undefined,
            "weight": "400 lbs",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
